src/sonic-py-common/tests/device_info_test.py

```diff
@@ -51,9 +53,16 @@
 
 
 class TestDeviceInfo(object):
-    @classmethod
-    def setup_class(cls):
-        print("SETUP")
+    @pytest.fixture(scope="class", autouse=True)
+    def sanitize_environment(self):
```

Comment:
it seems doing something like tox. https://tox.readthedocs.io/en/latest/index.html#basic-example

tox will take care of environment isolation for you: it will strip away all operating system environment variables not specified via passenv. Furthermore, it will also alter the PATH variable so that your commands resolve first and foremost within the current active tox environment. In general all executables in the path are available in commands, but tox will emit a warning if it was not explicitly allowed via allowlist_externals.

Reply:
True. Do we want to introduce tox to our build process?